src/migrator/migrator.ts

```diff
@@ -21,6 +22,24 @@ export type Migration = string | MigrationFn;
 
 export class MigrationError extends TypedError {}
 
+// Tagged template to convert binds from `?` format to the necessary output format,
+// eg `$1`/`$2`/etc for postgres
+const binds = (strings: TemplateStringsArray, ...binds: number[]) =>
```

Comment:
I don't think this works as intended or I miss something: given `S`, this will always return `S` as is for all strings I see in this PR.

Reply:
Cool, yeah, I had to look up about tagged templates and what I read didn't add up but I wasn't sure whether it was me not misunderstanding.